src/mlpack/methods/random_forest/CMakeCache.txt

```diff
@@ -0,0 +1,376 @@
+# This is the CMakeCache file.
```

Comment:
This is a generated file that we shouldn't add to the repository; can you remove it please? :+1:

Reply:
When I tried to build the binding these automatically get created.

Reply:
There's no need to add any generated files to the git repository---it's just fine that they get generated, but we shouldn't add them to the PR. :+1: